daily-challenges/2024-12/11/ducnq.k11.spec.ts

```
@@ -0,0 +1,24 @@
+// ## Yêu cầu
+// - Viết hàm `doiTien(soTien)` nhận vào số tiền cần đổi (đơn vị: đồng)
+// - Trả về một object chứa số lượng từng mệnh giá sao cho tổng số tờ tiền là ít nhất
+// - Các mệnh giá có sẵn: 500000, 200000, 100000, 50000, 20000, 10000, 5000, 2000, 1000
+// - Nếu không thể đổi được (số tiền < 1000), trả về `null`
+
+let menhGia = [500000, 200000, 100000, 50000, 20000, 10000, 5000, 2000, 1000];
+let result = {};
+
+function moneyExchange(amount:any){
+    if(amount < 1000){
+        return null;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'result' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)